_posts/2023-09-13-prospero-add-feature-pack.adoc

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title:  "Installing additional feature packs in WIldfly"
+date:   2023-09-13
+tags:   prospero galleon wildfly
+author: spyrkob
+description: Using Prospero to installing additional feature packs in existing Wildfly servers.
+---
+
+== Additional feature packs for Wildfly
+
+Wildfly server capabilities can be extended by installing additional feature packs such as wildfly-datasources-galleon-pack or wildfly-myfaces-galleon-pack.
+The feature packs can be installed using Galleon or Prospero.
+
+In this article we will use Prospero to install a MariaDB datasource provided by wildfly-datasources-galleon-pack.
+
+== Prerequisites
+
+1. Download and unzip Wildfly 29.0.1.Final from https://www.wildfly.org/downloads/
```

Comment:
We can also directly link to https://github.com/wildfly/wildfly/releases/download/29.0.1.Final/wildfly-29.0.1.Final.zip

Reply:
I assume from the rest of the article that the server must be copied to `/tmp/wildfly`. If that's the case, that better to specify it now. Or use the instructions to do so with the `cd /tmp/` & `wget`.